protelis/protelis-test/src/test/resources/method08.pt

```diff
@@ -0,0 +1,10 @@
+// EXPECTED_RESULT: 18
+
+import java.lang.String.format
+import java.util.Arrays.asList
+
+// Check call of over-length and minimum-length varargs method
+let l = asList(5,3,7,2,1); // Check call of varargs with multiple args
```

Comment:
I propose using Guava's [`List.newArrayList`](https://google.github.io/guava/releases/snapshot/api/docs/com/google/common/collect/Lists.html#newArrayList-E...-) which produces a new *modifiable* `List` with the same varargs invocation used by `Arrays.asList`. I believe it would make the test work and allow us to remove the `setAccessible(true)` hack on the reflective calls.

We already have Guava as a dependency, so the change should be pretty easy. What do you think @jakebeal? Was the workaround  in the interpreter inserted specifically for dealing with the lists produced by `asList`?